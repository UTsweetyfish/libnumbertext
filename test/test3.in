0
3
13
23
33
303
3003
