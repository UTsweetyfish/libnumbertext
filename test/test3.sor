__numbertext__
^0 zero
1 uno
2 due
^3$ tre
3$ trÃ©
3 tre
4 quattro
5 cinque
6 sei
7 sette
8 otto
9 nove
10 dieci
11 undici
12 dodici
13 tredici
14 quattordici
15 quindici
16 sedici
17 diciassette
18 diciotto
19 diciannove
2([18]) vent$1
2(\d) venti$1
3([18]) trent$1
3(\d) trenta$1
4([18]) quarant$1
4(\d) quaranta$1
5([18]) cinquant$1
5(\d) cinquanta$1
6([18]) sessant$1
6(\d) sessanta$1
7([18]) settant$1
7(\d) settanta$1
8([18]) ottant$1
8(\d) ottanta$1
9([18]) novant$1
9(\d) novanta$1
(\d)(\d\d) $1cento$2
1(\d{3}) mille$1
(\d{1,2})(\d{3}) $1mila$2
(\d{3})(\d{3}) $1mila $2
