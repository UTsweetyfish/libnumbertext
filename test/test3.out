zero
tre
tredici
ventitré
trentatré
trecentotré
tremilatré
